examples/kafka_spark_demo/feature_repo/streaming_features_tutorial.ipynb

```diff
@@ -0,0 +1,877 @@
+      "\tfound org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.0.0 in central\n",
+      "\tfound org.apache.kafka#kafka-clients;2.4.1 in central\n",
+      "\tfound com.github.luben#zstd-jni;1.4.4-3 in central\n",
+      "\tfound org.lz4#lz4-java;1.7.1 in local-m2-cache\n",
+      "\tfound org.xerial.snappy#snappy-java;1.1.7.5 in central\n",
+      "\tfound org.slf4j#slf4j-api;1.7.30 in local-m2-cache\n",
+      "\tfound org.spark-project.spark#unused;1.0.0 in central\n",
+      "\tfound org.apache.commons#commons-pool2;2.6.2 in central\n",
+      ":: resolution report :: resolve 374ms :: artifacts dl 13ms\n",
+      "\t:: modules in use:\n",
+      "\tcom.github.luben#zstd-jni;1.4.4-3 from central in [default]\n",
+      "\torg.apache.commons#commons-pool2;2.6.2 from central in [default]\n",
+      "\torg.apache.kafka#kafka-clients;2.4.1 from central in [default]\n",
+      "\torg.apache.spark#spark-sql-kafka-0-10_2.12;3.0.0 from central in [default]\n",
+      "\torg.apache.spark#spark-token-provider-kafka-0-10_2.12;3.0.0 from central in [default]\n",
+      "\torg.lz4#lz4-java;1.7.1 from local-m2-cache in [default]\n",
+      "\torg.slf4j#slf4j-api;1.7.30 from local-m2-cache in [default]\n",
+      "\torg.spark-project.spark#unused;1.0.0 from central in [default]\n",
+      "\torg.xerial.snappy#snappy-java;1.1.7.5 from central in [default]\n",
+      "\t---------------------------------------------------------------------\n",
+      "\t|                  |            modules            ||   artifacts   |\n",
+      "\t|       conf       | number| search|dwnlded|evicted|| number|dwnlded|\n",
+      "\t---------------------------------------------------------------------\n",
+      "\t|      default     |   9   |   0   |   0   |   0   ||   9   |   0   |\n",
+      "\t---------------------------------------------------------------------\n",
+      ":: retrieving :: org.apache.spark#spark-submit-parent-be0e71e9-46f4-47a3-a6fc-3661ae6b71c8\n",
+      "\tconfs: [default]\n",
+      "\t0 artifacts copied, 9 already retrieved (0kB/9ms)\n",
+      "22/06/17 11:32:59 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable\n",
+      "Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties\n",
+      "Setting default log level to \"WARN\".\n",
+      "To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "os.environ['PYSPARK_SUBMIT_ARGS'] = \"--packages=org.apache.spark:spark-sql-kafka-0-10_2.12:3.0.0 pyspark-shell\"\n",
+    "spark = SparkSession.builder.master(\"local\").appName(\"feast-spark\").getOrCreate()\n",
+    "spark.conf.set(\"spark.sql.shuffle.partitions\", 5)\n",
+    "\n",
+    "from typing import List"
```

Comment:
nit shouldn't this `from typing import List` import happen before the `os.environ` stuff

Reply:
fixed